visualizer/semanticsEditor.js

```diff
@@ -66,6 +101,12 @@
       opSignature += '(' + argValues.join(',') + ')';
     }
     var opNameContainer = block.appendChild(domUtil.createElement('operation'));
+    var semanticOperations = ohmEditor.semantics.getSemantics();
+    var operationCounts = Object.keys(semanticOperations.operations).length +
```

Comment:
Nit: Should probably be named "operationCount", not "operationCounts", since it's just one number.